assignments/prototype-refactor.js

```diff
@@ -7,3 +7,170 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+class GameObject {
+    constructor(attributes) {
+        this.createdAt = attributes.createdAt;
+        this.dimensions = attributes.dimensions;
+    }
+    destroy() {
+        return `${this.name} was removed from the game`;
+    }
+}
+
+class CharacterStats extends GameObject{
+    constructor(childAttributes) {
+        super(childAttributes);
+        this.hp = childAttributes.hp;
+        this.name = childAttributes.name;
```

Comment:
Here's another place where it might be more convenient:
```javascript
constructor(childAttributes) {
    const { hp, name } = childAttributes;
    super(childAttributes);
    this.hp = hp;
    this.name = name;
}
```